win32ss/user/user32/windows/dialog.c

```diff
@@ -2624,7 +2624,19 @@ IsDialogMessageW(
              if (!(dlgCode & DLGC_WANTARROWS))
              {
                  BOOL fPrevious = (lpMsg->wParam == VK_LEFT || lpMsg->wParam == VK_UP);
-                 HWND hwndNext = GetNextDlgGroupItem( hDlg, lpMsg->hwnd, fPrevious );
+                 HWND hwndFirst = NULL;
+                 HWND hwndNext = GetNextDlgGroupItem(hDlg, lpMsg->hwnd, fPrevious);
+
+                 /* Skip static elements when arrow-moving through a list of controls */
+                 /* hwndFirst to prevent infinite looping */
```

Comment:
```suggestion
                 /* Skip static elements when arrow-moving through a list of controls.
                  * Check with hwndFirst to prevent infinite looping. */
```